charts/argo-workflows/Chart.yaml

```diff
@@ -17,4 +17,4 @@ annotations:
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: fixed
-      description: Accept multi auth mode for server.
+      description: Add parameters for tuning revisionHistoryLimit and emptyDir volumes.
```

Comment:
I just wondered why `fixed`, maybe `added` ? 👀 
If `added` suites to this PR, please bump minor version instead of patch version.

Reply:
From the `CONTRIBUTING.md`:
> Please ensure chart version changes adhere to semantic versioning standards:
>
> * Major: Large chart rewrites, major non-backwards compatible or destructive changes
> * Minor: New chart functionality (sidecars), major application updates or minor non-backwards compatible changes
> * Patch: App version patch updates, backwards compatible optional chart features

I'd say they're compatible optional chart features as the defaults match the original values, (`revisionHistoryLimit` has an implicit default of 10 if not specified).

Reply:
Oh that's right.

What about below? 
>I just wondered why `fixed`, maybe `added` ?

Reply:
I assumed it had to match the conventional commit to imply a patch bump. If it's preferable to use `added` instead here, I'll update it.

Reply:
Since this PR is not for fixing bugs, so I wonder it's supposed to be `added` (or `changed` for params configurable). 😄

Reply:
Updated and rebased against main.